U/Usrsctp/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Usrsctp"
+version = v"0.9.5"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/sctplab/usrsctp.git", "07f871bda23943c43c9e74cc54f25130459de830")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/usrsctp/
+./bootstrap 
+./configure --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; )
+]
```

Comment:
Can we try and build for all platforms and see why they fail?
```suggestion
platforms = supported_platforms(; experimental=true)
```

Reply:
Sure thing :+1: